pkg/cmd/issue/view/view.go

```diff
@@ -256,7 +258,8 @@ func printHumanIssuePreview(opts *ViewOptions, issue *api.Issue) error {
 
 func issueStateTitleWithColor(cs *iostreams.ColorScheme, issue *api.Issue) string {
 	colorFunc := cs.ColorFromString(prShared.ColorForIssueState(*issue))
-	return colorFunc(strings.Title(strings.ToLower(issue.State)))
+	c := cases.Title(language.English)
+	return colorFunc(c.String(strings.ToLower(issue.State)))
```

Comment:
There are only two possible issue states; we could just hardcode their labels rather than generating them